nullaway/src/main/java/com/uber/nullaway/dataflow/AccessPath.java

```diff
@@ -255,6 +256,38 @@ public static AccessPath getAccessPathForNodeWithMapGet(Node node, @Nullable Typ
     }
   }
 
+  /**
+   * Gets corresponding AccessPath for a class field element.
+   *
+   * @param field The class field element.
+   * @param receivers The list of receivers.
+   * @return corresponding AccessPath.
+   */
+  public static AccessPath fromFieldAndBase(Element field, List<Element> receivers) {
```

Comment:
I would switch the order of these arguments, to match the order of the access path